supersuit/vector/markov_vector_wrapper.py

```diff
@@ -52,6 +52,13 @@ def step_wait(self):
         return self.step(self._saved_actions)
 
     def reset(self, seed=None, options=None):
+        if seed is None:
```

Comment:
I'm a bit confused, won't this pass the same seed to all of the environments, therefore, this does the opposite of what you want. 
Even so, this should be part of a second PR, not this one if possible

Reply:
I don't believe this will create the same seed. Each time `np.random.randint` is called, np internally updates its internal state, meaning that a new seed is created in the next call - [docs explaining this](https://jax.readthedocs.io/en/latest/jax-101/05-random-numbers.html#random-numbers-in-numpy). 

This would only create the same seed for all envs if `np.random.seed` is used in each process to set them all to have the same seed. 

Nonetheless, I agree this should be removed from this PR.